src/main/java/org/sciborgs1155/robot/Intake/SimIntake.java

```diff
@@ -0,0 +1,21 @@
+package org.sciborgs1155.robot.Intake;
+
+import static org.sciborgs1155.robot.Constants.*;
+
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+
+public class SimIntake implements IntakeIO {
+  DCMotorSim motor = new DCMotorSim(DCMotor.getNEO(1), 1, 1);
+
+  @Override
+  public double getSpeed() {
+    return motor.getAngularVelocityRadPerSec();
```

Comment:
also, make sure you're consistent about the units in sim and real! If you're using radians per second in sim, do that in real too.  Or you can use m/s in both. whatever you do, just make sure it is:
a) consistent between sim and real
b) consistent between subsystems (intake, hopper, shooter)
c) articulated in the comments (ask me/look up how to do javadocs comments)